x-pack/plugins/cases/public/components/recent_cases/loading_placeholders.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiLoadingContent, EuiSpacer } from '@elastic/eui';
+import React from 'react';
+
+const LoadingPlaceholdersComponent: React.FC<{
+  lines: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
+  placeholders: number;
+}> = ({ lines, placeholders }) => (
+  <>
+    {[...Array(placeholders).keys()].map((_, i) => (
```

Comment:
nit: `[...Array(placeholders).keys()].map` -> `new Array(placeholders).keys().map`

Reply:
eh, it aint broke dont fix it